src/Core/Algorithms/BrainStimulator/ElectrodeCoilSetupAlgorithm.cc

```diff
@@ -792,7 +792,7 @@ boost::tuple<DenseMatrixHandle, FieldHandle, FieldHandle, VariableHandle> Electr
 
         if (compute_third_output) /// a lot of code is inside this if (make sure it creates proper results)
         {
-          BBox proto_bb = prototype->vmesh()->get_bounding_box();
+          AxisAlignedBBox proto_bb = prototype->vmesh()->get_bounding_box();
```

Comment:
Here's where `auto` would've been your friend. With it, future return-type changes do not force all these call-site updates.